ee/tasks/hooks.py

```diff
@@ -1,15 +1,25 @@
 import json
+from urllib.parse import urlparse
 
 import requests
 from celery.task import Task
 from django.core.serializers.json import DjangoJSONEncoder
 
 
+def valid_domain(url) -> bool:
```

Comment:
I think we should rather do it in `HookSerializer`'s so that instead of mysteriously never sending anything, the API clearly returns a "hooks.zapier.com is the only allowed target domain" 400 error. `HookSerializer` already has `validate_event` – analogously we can do this check on `target` by adding validation method `validate_target`, which Django REST Framework will run automatically on creates/updates.